src/nexuscli/cli.py

```diff
@@ -238,6 +242,31 @@ def cmd_upload(args):
     return 0
 
 
+def cmd_download(args):
+    """Performs ``nexus3 download``"""
+    nexus_client = get_client()
+    source = args['<from_repository>']
+    destination = args['<to_dst>']
+
+    sys.stderr.write(
+        'Downloading {source} to {destination}\n'.format(**locals()))
+
+    FLATTEN = args.get('--flatten')
```

Comment:
UPPERCASE are used for constants. Kind of weird to see these use-once variables here.

How about doing the `args.get()` directly as a kw argument of `download()`?  Break some lines - that's fine.